app/code/Magento/InventoryCatalog/view/adminhtml/web/js/product/components/stock-info.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * @api
+ */
+define([
+    'underscore',
+    'Magento_Ui/js/form/element/abstract'
+], function (_, Abstract) {
+    'use strict';
+
+    return Abstract.extend({
+        defaults: {
+            template: 'Magento_InventoryCatalog/product/stock-info'
+        },
```

Comment:
Make here import of stocks from data source. Example:
```js
return Abstract.extend({
    defaults: {
        template: 'Magento_InventoryCatalog/product/stock-info',
        imports: {
            stocks: '${ $.provider }:data.stocks'
        }
    },
}
```

And use this stocks inside getStocksQtyInfo, like `this.stocks`